api/serializers/phones.py

```diff
@@ -38,3 +38,27 @@ class Meta:
             "id",
             "location",
         ]
+
+
+class InboundContactSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = InboundContact
+        fields = [
+            "id",
+            "relay_number",
+            "inbound_number",
+            "last_inbound_date",
+            "num_calls",
+            "num_calls_blocked",
```

Comment:
Do we want `num_calls_blocked` and `num_texts_blocked` to be API writable?

Reply:
Not right now - there's no reason for an external client to change these values. Only the incoming call or incoming text logic should update these values.

Reply:
>> Do we want `num_calls_blocked` and `num_texts_blocked` to be API writable?

> Not right now - there's no reason for an external client to change these values. Only the incoming call or incoming text logic should update these values.

OK, please add `num_calls_blocked` and `num_texts_blocked` to `read_only_fields`